app/src/main/java/com/jalasoft/routesapp/data/model/remote/TransportMethod.kt

```diff
@@ -0,0 +1,15 @@
+package com.jalasoft.routesapp.data.model.remote
+
+import android.location.Location
+
+data class TransportMethod(
+    val name: String,
+    val lines: List<Route>
+)
+
+data class Route(
```

Comment:
Same as the other one, we're not using it, remove it :)

Reply:
Removed